AffirmSDK/AffirmPopupViewController.m

```diff
@@ -34,21 +34,4 @@ - (void)viewDidLoad
     [self.webView loadRequest:[[NSURLRequest alloc] initWithURL:self.startURL]];
 }
 
-#pragma mark - WKNavigationDelegate
-
-- (void)webView:(WKWebView *)webView decidePolicyForNavigationAction:(WKNavigationAction *)navigationAction decisionHandler:(void (^)(WKNavigationActionPolicy))decisionHandler
```

Comment:
What is the intent and logic for this method? Just making sure the decisionHandler logic is correct? "**WKNavigationActionPolicyCancel**" when canOpenUrl returns true, and "**WKNavigationActionPolicyAllow**" when canOpenUrl return false